man/systemd-pcrlock.xml

```diff
@@ -0,0 +1,560 @@
+<?xml version="1.0"?>
+<!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
+<refentry id="systemd-pcrlock" xmlns:xi="http://www.w3.org/2001/XInclude" conditional='ENABLE_BOOTLOADER'>
+
+  <refentryinfo>
+    <title>systemd-pcrlock</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>systemd-pcrlock</refentrytitle>
+    <manvolnum>1</manvolnum>
```

Comment:
```suggestion
    <manvolnum>8</manvolnum>
```

Reply:
Either that, or the newly introduced service units need to reference `man:systemd-pcrlock(1)` in `Documentation=`.